acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/tls/ClientSideTlsAcceptanceTest.java

```diff
@@ -77,8 +81,7 @@ private Signer createAndStartSigner(
       final int downstreamWeb3Port,
       final int listenPort,
       final Path workDir)
-      throws IOException, CertificateException, NoSuchAlgorithmException, KeyStoreException,
-          UnrecoverableKeyException {
+      throws IOException, CertificateException, NoSuchAlgorithmException, KeyStoreException {
```

Comment:
nit: no real benefit in multiple throws for a test. just throw exception here and below?